api/v1/routes/subscription.py

```diff
@@ -0,0 +1,78 @@
+
+from fastapi import status, APIRouter
+from fastapi.responses import JSONResponse
+from fastapi.encoders import jsonable_encoder
+from api.v1.models.subscription import Subscription
+from api.v1.models.user import User 
+from typing import Annotated
+from api.utils.dependencies import get_current_user
+from api.db.database import get_db
+from sqlalchemy.orm import Session
+from fastapi import Depends
+from api.v1.schemas.subscription import SubscriptionRequest
+
+db = next(get_db())
+subscription = APIRouter(prefix="/api/v1/user/subscription", tags=["Subscription"])
+
+def send_cancellation_confirmation_notification(subscription: Subscription, db: Session):
+    try:
+        subscription_user = db.query(User).filter_by(id=subscription.user_id).first()
+        return JSONResponse(
+                    status_code=status.HTTP_200_OK,
+                    content=jsonable_encoder({
+                    "message": "Sent Subscription Cancellation Confirmation notification successfully",
+                    "name": subscription_user.username,
+                    "subscription": {
+                        "id": subscription.id,
+                        "plan": subscription.plan,
+                        "cancellation_date": subscription.end_date
+                    }
+                }),) 
+
```

Comment:
There's `JsonResponseDict` class in the `utils.json_response` module that handles this for you according to the specified convention, check it out and update all round

Reply:
I've updated it